FindTown/FindTown/Presentation/MyPageScene/MyPage/MyPageViewController.swift

```diff
@@ -101,7 +101,9 @@ final class MyPageViewController: BaseViewController {
             .subscribe { [weak self] _ in
                 self?.showAlertSuccessCancelPopUp(title: "동네한입", message: "로그아웃 하시겠어요?",
                                                   successButtonText: "로그아웃", cancelButtonText: "취소",
-                                                  successButtonAction: { })
+                                                  successButtonAction: {
+                    self?.viewModel?.logout()
+                })
```

Comment:
```suggestion
                                                  successButtonAction: {
                    self?.indicator.startAnimating()
                    self?.indicator.isHidden = false
                    self?.viewModel?.logout()
                })
```

큰 건 아니지만.. 로그아웃할 때 잠깐 서버와 통신하는 그 순간이라도 indicator를 보여주면 어떨까요?
이미 indicator는 mypage에서 사용중이니  저 2줄만 넣으면 됩니다..! + 회원탈퇴도.. ㅎㅎㅎㅎ; 😀

Reply:
오오 넵 적용했습니다! 감사합니다 👍